src/Client.js

```diff
@@ -610,6 +610,43 @@ class Client extends EventEmitter {
         }, chatId);
     }
 
+    /**
+     * Enables and returns the pin state of the Chat
+     * @returns {boolean}
+     */
+    async pinChat(chatId) {
+        return await this.pupPage.evaluate(async chatId => {
+            let chat = await window.Store.Chat.get(chatId);
+            if (chat.pin) {
+                return true;
+            }
+            const MAX_PIN_COUNT = 3;
+            if (window.Store.Chat.models.length > MAX_PIN_COUNT) {
+                let maxPinned = await window.Store.Chat.models[MAX_PIN_COUNT - 1].pin;
```

Comment:
```suggestion
                let maxPinned = window.Store.Chat.models[MAX_PIN_COUNT - 1].pin;
```

No need for `await` here.